demos/999-storage-import-vhd-blob/prereqs/CreateWritableBlobContainerUri.ps1

```diff
@@ -0,0 +1,25 @@
+param(
```

Comment:
remove this file and use listAccountSas() in the template to get the writable uri

Reply:
Found an example in the quickstart https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.web/web-app-diagnostics-logs-blob-container/azuredeploy.json

It looks great. The approach is to harcode the start and expiry dates for a large period.

Maybe it is not important here for the prereqs but it would be cool to have set now for start and now + a paremeter (for example 4hours) for the expiry date.

Reply:
It works well. I have set a static 4 months validity (until nov. 2021). 

The pipeline just raises this error in the best practices test:
[-] Outputs Must Not Contain Secrets (8 ms) 
##[error]        Output contains secret

Reply:
The output contains secret it is the same issue as in the storage-import-zipped-vhds PR #9561. When completely fixed there i can adapt here with the same method

Reply:
The BP error is coming from the secret in the output - that needs to be fixed.  You can either put the secret in keyvault (which means you have another prereq) or generate the token in the maintemplate.

For expiration, you need to make it a delta from the current date, otherwise this will start failing in 4 months.  See: https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions#date-functions

Reply:
Thanks, I have updated to use securely pass prereqs output to the maintemplate using Keyvault and also fixing delta time.

So now there is just the little glitches that remain in BP and Deploy mainTemplae - same as in PR #9561 due to the .reference.keyVault.id parameter filled with GET-PREREQ-KeyVaultId